hail_scripts/v02/update_models/update_mt_schema.py

```diff
@@ -0,0 +1,53 @@
+import hail as hl
+
+from lib.model.base_mt_schema import BaseMTSchema, row_annotation, RowAnnotationOmit
+from hail_scripts.v02.utils.computed_fields import variant_id
+
+class UpdateSchema(BaseMTSchema):
```

Comment:
If the annotations are as simple as this, I actually don't think the schema is needed. The schema models were made to model complex relationships between the data, but if most of this is a few annotations that are used only in the script, we can just add it straight in the script.

Reply:
Yeah, just code reuse seems like a good idea for me. When you see how the whole index load happens and understand it, update index code would look obvious right away.